api/index.js

```diff
@@ -0,0 +1,6 @@
+const collective = require('./collective');
+const campaign = require('./campaign');
+const topic = require('./topic');
+const user = require('./user');
+
+module.exports = Object.assign(collective, campaign, topic, user);
```

Comment:
If you change this to be:

```javascript
module.exports = {
  collective: require('./collective'),
  campaign: require('./campaign'),
  topic: require('./topic'),
  user: require('./user')
};
```

We'll get better type inference from VS code for free.

Reply:
cool